packages/ember-template-compiler/tests/system/bootstrap-test.js

```diff
@@ -69,57 +62,42 @@ QUnit.test('template without data-template-name or id should default to applicat
   checkTemplate('application');
 });
 
-if (typeof Handlebars === 'object') {
```

Comment:
I'm pretty sure this is not supported any longer, as the string `x-raw-handlebars` appears nowhere in the codebase but here.